ui/src/api/mixins/TableCollection.ts

```diff
@@ -0,0 +1,19 @@
+import { Collection } from 'alcaeus'
+import * as ns from '@cube-creator/core/namespace'
+import { schema } from '@tpluscode/rdf-ns-builders'
+import { findOperation } from '../common'
+import { TableCollection } from '@/types'
+
+export type Constructor = new (...args: any[]) => Collection;
+
+export default function Mixin<Base extends Constructor> (base: Base) {
+  return class extends base implements TableCollection {
+    get actions () {
+      return {
+        create: findOperation(this, schema.CreateAction),
+      }
+    }
+  }
+}
+
+Mixin.appliesTo = ns.cc.TableCollection
```

Comment:
Not to change this now but wonder if you really need this as a mixin. Food for thought if we indeed use `schema` for common actions

Reply:
Yeah I'm not doing anything else that extracting the operations in `actions` at the moment so if we use schema for all of the operations, I can have a generic mixin for collections.